ui/apps.py

```diff
@@ -21,3 +21,7 @@ def ready(self):
         if missing_settings:
             raise ImproperlyConfigured(
                 'The following settings are missing: {}'.format(', '.join(missing_settings)))
+
+        # write the x509 certification & key to files
+        from ui.utils import write_x509_files
+        write_x509_files()
```

Comment:
I talked with Giovanni about checking for existing files first before writing them; decided instead it might be better to just overwrite them on startup in case the x509 keys/certs change.